frontend/src/app/components/ExperimentPage.js

```diff
@@ -338,6 +360,15 @@ export default class ExperimentPage extends React.Component {
     }, 5000);
   }
 
+  clickUpgradeNotice() {
```

Comment:
@chuckharmston i don't think there's a race condition here b/c the link has `target=_blank"